### DISCUSSION TITLE
OpenBB Terminal v3.2.5

### DISCUSSION THREAD

**Discussion by jmaslek** [OpenBB-finance/OpenBB#6219] [General]
This will be the final release of the OpenBB Terminal.

You can read about this decision and what comes next [here](https://openbb.co/blog/sunsetting-openbb-terminal-why-how-and-what-now).

This version has fixed some of the previous bugs that were related to dependencies and has removed a chunk of deprecated dependencies.

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenBB-finance/OpenBBTerminal/releases/tag/v3.2.5'>OpenBB Terminal v3.2.5</a>.</em>

**Comment by DanieliteHazel** [comment]
A beautiful Gm to everyone! 

My name is Daniel Hazel. I'm not a developer but I will love to be one. But currently a community moderator and manager. I will be happy if there is anyone here who will be willing to take me on their project. 

It will be really nice working with the person, I can cover any shifts at all. I can set all telegram bot and I'm very hardworking. 

Thank you all for having me. Looking forward to working with lots of you. 

Have a pleasant day everyone!
